pydicom/dicomdir.py

```diff
@@ -68,7 +68,7 @@ def __init__(self,
                 raise InvalidDicomError(msg)
         if is_implicit_VR or not is_little_endian:
             msg = ('Invalid transfer syntax for DICOMDIR - '
-                   'Implicit Little Endian expected.')
+                   'Explicit Little Endian expected.')
```

Comment:
The test on line 69 is `if is_implicit_VR` so only implicit can hit this message.